CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## CHANGELOG
 
+### 6.1
```

Comment:
Is this targeted towards 6.0.0 or 6.1.0?

Reply:
My assumption was 6.1 as 6.0 should be nominally closed to these kind of changes.